ingestion/src/metadata/ingestion/source/sample_data.py

```diff
@@ -279,7 +268,7 @@ def create(cls, config_dict, metadata_config_dict, ctx):
         return cls(config, metadata_config, ctx)
 
     def prepare(self):
-        self.policies_source.prepare()
```

Comment:
This is no longer needed. 

The seed data during application start is sufficient